assets/src/edit-story/utils/isLinkBelowLimit.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { FULLBLEED_RATIO, PAGE_HEIGHT, PAGE_WIDTH } from '../constants';
+
+const FULLBLEED_HEIGHT = PAGE_WIDTH / FULLBLEED_RATIO;
+const DANGER_ZONE_HEIGHT = (FULLBLEED_HEIGHT - PAGE_HEIGHT) / 2;
+
+const getYCoordinatesByAngle = ({ y, width, height, rotationAngle }) => {
```

Comment:
Note: we only need the `y` coordinates here since we're just checking the below 20% level only, `x` is not relevant for now. Will add a comment about it.

Reply:
This seems like a lot of math to do "by hand". Can't we utilize some util library here? Not that I doubt your l33t math skillz 😜

Reply:
There are some geometry libraries that would work for that, however, these usually come with a lot of other things, not sure if it makes sense to add a bunch of kB just for the purpose of making this small calculation.
The calculation was quite similar to what we already had in the codebase so refactored it to use that instead.